src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -184,38 +186,35 @@ def configure_container_dict(
     else:
         logger.debug(f"container_dict before processing: {container_dict}")
 
-    if not host_mount_path:
-        if file_transfer_service:
-            host_mount_path = pyfluent.USER_DATA_PATH
-        elif os.getenv("PYFLUENT_HOST_MOUNT_PATH", None):
-            host_mount_path = pyfluent.EXAMPLES_PATH
-        elif pyfluent.CONTAINER_MOUNT_PATH:
-            host_mount_path = pyfluent.CONTAINER_MOUNT_PATH
```

Comment:
I'm not sure why host path is set to container path. I've removed it for now.

Reply:
Found this was used for unittests in github CI, I'm now setting both source/mount to `EXAMPLES_PATH` for github CIs.